date: '13.12.2021'
sections:
  security_fixes:
    - '{% octicon "alert" aria-label="The alert icon" %} **Критический.** Уязвимость удаленного выполнения кода в библиотеке Log4j, идентифицированная как [CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228), повлияла на все версии {% data variables.product.prodname_ghe_server %}, предшествующие 3.3.1. Библиотека Log4j используется в службе с открытым кодом, выполняющейся на экземпляре {% data variables.product.prodname_ghe_server %}. Эта уязвимость исправлена в версиях {% data variables.product.prodname_ghe_server %} 3.0.22, 3.1.14, 3.2.6 и 3.3.1.  Дополнительные сведения см. в [этой публикации](https://github.blog/2021-12-13-githubs-response-to-log4j-vulnerability-cve-2021-44228/) в блоге GitHub.'
    - '**Обновление от 17 декабря 2021 года**. Исправления для этого выпуска также устраняют проблему [CVE-2021-45046](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046), опубликованную после выпуска. Чтобы устранить проблемы CVE-2021-44228 и CVE-2021-45046, не требуется никаких дополнительных обновлений {% data variables.product.prodname_ghe_server %}.'
  known_issues:
    - "Злоумышленник мог создать первого пользователя с правами администратора на только что настроенном сервере {% data variables.product.prodname_ghe_server %} без пользователей."
    - Настраиваемые правила брандмауэра удаляются в процессе обновления.
    - "Отслеживаемые файлы Git LFS, [отправленные через веб-интерфейс](https://github.com/blog/2105-upload-files-to-your-repositories), ошибочно добавлялись напрямую в репозиторий."
    - "Проблемы нельзя закрыть, если они содержат постоянную ссылку на большой двоичный объект в том же репозитории, где путь к файлу большого двоичного объекта превышает 255 символов."
    - "Когда в GitHub Connect включен параметр Users can search GitHub.com (Пользователи могут выполнять поиск на GitHub.com), результаты поиска на GitHub.com не включают в себя проблемы в частных и внутренних репозиториях."
    - "Реестр npm {% data variables.product.prodname_registry %} больше не возвращает значение времени в ответах метаданных. Это изменение позволяет существенно улучшить производительность. Мы продолжим хранить все данные, необходимые для возврата значения времени в качестве части ответа метаданных, и возобновим возврат этого значения в будущем, как только решим существующие проблемы с производительностью."
    - "Ограничения ресурсов, относящиеся к обработке перехватчиков перед получением, могут приводить к сбою некоторых из них."
    - '{% data reusables.release-notes.ghas-3.4-secret-scanning-known-issue %}'
